getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -297,8 +297,8 @@ the ``_process`` function (make sure it's not indented under the `else`):
 
         Position += velocity * (float)delta;
         Position = new Vector2(
-            x: Mathf.Clamp(Position.x, 0, ScreenSize.x),
-            y: Mathf.Clamp(Position.y, 0, ScreenSize.y)
+            x: Mathf.Clamp(Position.X, 0, ScreenSize.X),
+            y: Mathf.Clamp(Position.Y, 0, ScreenSize.Y)
```

Comment:
What about the two `Vector2` fields, shouldn't they be capitalized here too?

Reply:
@akien-mga These are "just" variable names `new Vector2(x: ..., y: ...)`. Capitalizing them would actually create an issue:
<img width="205" alt="Bildschirmfoto 2023-03-06 um 11 47 02" src="https://user-images.githubusercontent.com/11627131/223088988-351c50c7-b3a0-4e5f-bf79-3bfb3992a12e.png">